ChemoTracker/components/Login/Login.js

```diff
@@ -26,39 +23,45 @@ export default class Login extends React.Component {
   }
 
   async onSubmit() {
-    this.setState({ errorLabel: '' });
-    if (this.state.loginUsername === '') {
-      this.setState({ errorLabel: 'Username is required' });
+    const api = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
```

Comment:
you can make the api call in services folder as what i made for symptom history :) 
i have the api constant there and the login api call

Reply:
i need this to switch to next screen on success though

Reply:
please see Calendar.js, you can see i imported the function there(line 12), and call the function with .then on line 56 of Calendar.js

Reply:
line 37-50 can be used as example too

Reply:
okies i'll refactor it

Reply:
can't get it to work. it gives me undefined even though it logs in